Dockerfile.ci-rp

```diff
@@ -32,10 +32,6 @@ WORKDIR /app
 ENV GOPATH=/root/go
 ENV GOFLAGS="-tags=containers_image_openpgp,exclude_graphdriver_btrfs,exclude_graphdriver_devicemapper"
 
-# Install golangci-lint and verify
-RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b /usr/local/bin v1.56.2 && \
-    golangci-lint --version || (echo "golangci-lint not found" && exit 1)
```

Comment:
Related to the discussion on container install methods - this concerns me. I chose this route specifically because golangci-lint's own documentation says that `go install` is unreliable: https://golangci-lint.run/welcome/install/#install-from-source - their reasoning:

```
go install/go get installation isn't recommended because of the following points:

    Some users use -u flag for go get, which upgrades our dependencies. Resulting configuration wasn't tested and isn't guaranteed to work.
    [go.mod](https://github.com/golangci/golangci-lint/blob/master/go.mod) replacement directive doesn't apply. It means a user will be using patched version of golangci-lint if we use such replacements.
    We've encountered a lot of issues with Go modules hashes.
    It allows installation from master branch which can't be considered stable.
    It's slower than binary installation.
```

Reply:
if we're ok with accepting this risk for all the other benefits we get with bingo, I can be onboard, just wanted to call this out.

Reply:
https://github.com/bwplotka/bingo?tab=readme-ov-file#real-life-examples has some information about it -- basically, bingo side-steps the issues that it doesn't recommend it because of.

Plus, the module pinning that bingo uses is better than downloading binaries, in terms of security, imo.